app/models/group/abteilung.rb

```diff
@@ -73,6 +77,15 @@ class Group::Abteilung < Group
   # Can't use validates :geolocations, length: { ... } because it counts in the
   # nested records that are #marked_for_destruction?
   validate :assert_geolocation_count
+  validate do
+    is_valid = geolocations.all? do |geolocation|
+      GEOLOCATION_SWITZERLAND_SOUTH_LIMIT < geolocation.lat.to_f \
+      && geolocation.lat.to_f < GEOLOCATION_SWITZERLAND_NORTH_LIMIT \
+      && GEOLOCATION_SWITZERLAND_WEST_LIMIT < geolocation.long.to_f \
+      && geolocation.long.to_f < GEOLOCATION_SWITZERLAND_EAST_LIMIT
+    end
+      errors.add(:base, :geo_not_in_switzerland) if !is_valid
```

Comment:
```suggestion
    errors.add(:base, :geo_not_in_switzerland) if !is_valid
```
Seltsam dass da RuboCop nicht ausschlägt.